src/modules/TrackAddition/components/TrackAddition/TrackAddition.tsx

```diff
@@ -152,6 +154,19 @@ const TrackAddition: React.FC = memo(() => {
     }
   }, [debounceValue]);
 
+  const clearValueHandler = () => {
+    setValue('url', '');
+  };
+
+  const getClipboardText = async () => {
+    navigator.permissions.query({ name: 'clipboard-read' });
```

Comment:
```suggestion
    navigator.permissions.query({ name: 'clipboard-read' as PermissionName });
```
добавь вот это, чтобы ts не ругался
https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1245